assets/js/components/adminbar/AdminBarApp.js

```diff
@@ -64,6 +66,15 @@ export default function AdminBarApp() {
 								? decodeHTMLEntity( currentEntityTitle )
 								: currentEntityURL
 							}
+							<p className="googlesitekit-adminbar__title--date-range">
+								{
+									sprintf(
+										/* translators: %s: number of days */
+										_n( 'over the last %s day', 'over the last %s days', dateRangeLength, 'google-site-kit', ),
+										28,
```

Comment:
Why is `28` hardcoded here? I know right now we always use the last `28` days for this data, but if we changed it there'd be a mismatch between the `_n()` string used… and also it would still show `28` 😅 

```suggestion
										dateRangeLength,
```